build/yamls/antrea-aks.yml

```diff
@@ -1353,7 +1353,7 @@ data:
     # Enable Antrea ClusterNetworkPolicy feature to complement K8s NetworkPolicy for cluster admins
     # to define security policies which apply to the entire cluster, and Antrea NetworkPolicy
     # feature that supports priorities, rule actions and externalEntities in the future.
-    #  AntreaPolicy: false
+      AntreaPolicy: true
```

Comment:
`make manifest` and re-push

Reply:
this is a temporary change for testing the PR, it will be reverted before merging. The toggle from `AntreaPolicy: false` to `AntreaPolicy: true` will be in a dedicated PR as it is an important feature graduation change: https://github.com/vmware-tanzu/antrea/pull/1999

Reply:
I see, thanks